cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -107,30 +112,35 @@ def test_tenant_quota_enforce_via_lifecycle_cloud(request, appliance, provider,
 # second arg is a list of lists, with each one a test is to be generated
 # sequence is important here
 # indirect is the list where we define which fixtures are to be passed values indirectly.
-@pytest.mark.skip('Catalog items are converted to collections. Refactoring is required')
 @pytest.mark.parametrize(
-    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg'],
+    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg', 'context'],
```

Comment:
**REQUIRED:** This looks rather messy seeing all the duplicated lines. The better way to do this is to use a second paramtetrize
```python
@pytest.mark.parametrize('context', ['ViaSSUI', 'ViaUI'])
@pytest.mark.parametrize(
    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg'],
     [
        [('cpu', 2), {}, ''],
        [('storage', 0.001), {}, ''],
        [('memory', 2), {}, ''],
        [('vm', '1'), {'catalog': {'num_vms': '4'}}, '###']
     ])
def test_something(context, set_roottenant_quota, custom_prov_data, extra_msg):
```

Reply:
The IDs don't need changing then either